place-work/v1alpha1/doc.go

```diff
@@ -0,0 +1,9 @@
+// Package v1 contains API Schema definitions for the place-work v1 API group
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=open-cluster-management.io/api/place-work
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+
+// +kubebuilder:validation:Optional
+// +groupName=work.open-cluster-management.io
```

Comment:
if the group name is work, it is better to put into work dir.

Reply:
Will move it to work.ocm.io